frontend/fingertips-frontend/package.json

```
@@ -5,17 +5,21 @@
   "scripts": {
     "dev:local-api": "concurrently \"npm run api:start\" \"next dev\" && npm run api:stop",
     "dev": "cross-env MOCK_SERVER=true next dev",
+    "dev-docker": "cross-env next dev",
     "generate:api-client": "openapi-generator-cli generate",
     "generate:ft-mocks": "npx msw-auto-mock ./../../api/definition/swagger.yaml -o ./mock/server",
     "build": "next build",
     "start": "next start",
     "api:start": "docker compose --profile api up --build",
     "api:stop": "docker compose --profile api down",
+    "start:all": "docker compose --profile all up --build --remove-orphans --wait -d",
+    "stop:all": "docker compose --profile all down",
     "lint": "next lint",
     "typecheck": "tsc",
     "test": "jest",
     "test-ci": "jest --ci --silent",
     "test-e2e": "playwright test",
+    "test-e2e-local-docker": "START_DOCKER_WEBSERVER=true npm run start:all && playwright test || true && npm run stop:all",
```

The `|| true` in the command could potentially mask errors in playwright test. If the test process crashes or is interrupted, the cleanup may not execute. Would it be a better option to pass this off to a dedicated nodejs script e.g. `scripts/run-e2e-tests.js` and have some more logic in there to handle the execution of each step?

---

interesting - ill investigate thanks

---

agreed in slack - https://bjss.slack.com/archives/C085Q20RZU0/p1738685611191999?thread_ts=1738681757.832509&cid=C085Q20RZU0 that we would see how this goes